test_config.yaml

```diff
@@ -6,11 +6,14 @@ database:
   password:
 
 server:
+  port: 8000
```

Comment:
same comment here. That was for mac only because macports is using these ports. Maybe it would be best to just put a warning in the mac section saying that because of macports you might want to run skyportal on another port (and add it both to config.yaml and test_config.yaml) or turn off macports.

Reply:
i will update the docs with a message about checking for port availability, thanks!